java8/src/main/java/com/dslplatform/json/processor/ConverterTemplate.java

```diff
@@ -109,10 +117,32 @@ private void asFormatConverter(final StructInfo si, final String name, final Str
 			code.append("\t\tprivate final java.lang.reflect.Type[] actualTypes;\n");
 		}
 
+		Map<String, TypeMirror> genericAttributes = new HashMap<>();

```

Comment:
It would be more consistent to do this analysis in Analysis.java (in analyze() method)

Reply:
I see, I will move this analysis to the Analysis class.